### PR TITLE
Updating documentation as per issue #376

### PR DESCRIPTION
**Pull request by zklosko**
#376 Updating more screenshots and unchecked boxes

### CODE REVIEW

**Comment by zklosko** [comment]
#376 Updating more screenshots and unchecked boxes

**Comment by zklosko** [comment]
@Robbt Waiting for your review and merge

File: docs/install.md [hunk 1]
```diff
@@ -1,26 +1,76 @@
 # Installing LibreTime
-There two methods of installing LibreTime - [Source](#source) or
-[Ubuntu package](#ubuntu-package).
+There two methods of installing LibreTime - [Source (recommended)](#source) or
+[Ubuntu package (experimental)](#ubuntu-package).
```
Comment (paddatrapper) [inline docs/install.md:3]: I'm not sure the Ubuntu package is experimental at this point. It is tested and works without issue. The only issue is that it supports a subset of the distros and versions that LibreTime supports (namely Ubuntu 16.04 and 18.04)

File: docs/manual/podcasts/index.md [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
 
 In order to add a podcast you need to get the RSS feed. All podcasts available on iTunes have a RSS feed but it is sometimes hidden. See this issue on our github page [#510](https://github.com/LibreTime/libretime/issues/510) for more information. RSS feeds that do not end in *.xml* may be accepted by LibreTime but might fail to download episodes; in that case, download the episode using a podcast client such as [gpodder](https://gpodder.github.io/) and then manually upload and schedule the episode. Podcast feeds coming from Anchor.fm have been known to have this issue.
 
-The podcast interfaces provides you with the ability generate [Smartblocks](../smartblocks) that can be used in conjunction with [Autoloading Playlists](../calendar/#autoloading-playlist) to schedule the newest episode of a podcast without human intervention.
+The podcast interfaces provides you with the ability to generate [Smartblocks](../smartblocks) that can be used in conjunction with [Autoloading Playlists](../calendar/#autoloading-playlist) to schedule the newest episode of a podcast without human intervention.
 
 ##Video Tutorials##
 Coming Soon
```
Comment (paddatrapper) [inline docs/manual/podcasts/index.md:10]: The video tutorials are here, so this Coming Soon can be removed

File: docs/install.md (hunk 1 above)
Comment (zklosko) [reply docs/install.md:3]: The last time I tried to use the Ubuntu package for installing, it failed, so I found an installation video on YouTube that recommended installing from source and it worked really well. I'll change it.

File: docs/manual/podcasts/index.md (hunk 2 above)
Comment (zklosko) [reply docs/manual/podcasts/index.md:10]: I found this one: https://www.youtube.com/watch?v=g-4UcD8qvR8. Are there others?

Comment (paddatrapper) [reply docs/manual/podcasts/index.md]: Thought there were more, but looking again, that one was all I could find too.

Comment (zklosko) [reply docs/manual/podcasts/index.md]: It's a start...

**Comment by zklosko** [comment]
Thanks @paddatrapper

**Pull request merged**